api/v1/storagecluster_types.go

```diff
@@ -137,11 +140,41 @@ type ManageCephObjectStoreUsers struct {
 	ReconcileStrategy string `json:"reconcileStrategy,omitempty"`
 }
 
+// ExternalStorageKind specifies a kind of the external storage
+type ExternalStorageKind string
+
+const (
+	// KindOCS specifies a "ocs" kind of the external storage
+	KindOCS ExternalStorageKind = "ocs"
+
+	// KindRHCS specifies a "rhcs" kind of the external storage
+	KindRHCS ExternalStorageKind = "rhcs"
+)
+
 // ExternalStorageClusterSpec defines the spec of the external Storage Cluster
 // to be connected to the local cluster
 type ExternalStorageClusterSpec struct {
-	// +optional
+	//+kubebuilder:default:=true
+	// Enable enables the external mode
 	Enable bool `json:"enable,omitempty"`
+
+	//+kubebuilder:default:=rhcs
+	// StorageProviderKind describes the kind of the external storage
+	StorageProviderKind ExternalStorageKind `json:"storageProviderKind,omitempty"`
+
+	// ConnectionString describes the blob of provider
+	ConnectionString string `json:"connectionString,omitempty"`
+
+	//+kubebuilder:default:="1Ti"
+	// RequestedCapacity defines the requested storage on the provider
```

Comment:
Should be a user friendly message. This is shown when we do `oc explain storagecluster.spec`.

Reply:
done